styles.css

```diff
@@ -147,7 +152,24 @@ main {
   margin-left: 1%;
   flex-direction: column;
   box-shadow:  5px 5px 10px #b3b3b3,
-             -5px -5px 10px #ffffff;
+  -5px -5px 10px #ffffff;
+}
+
+.card-div {
+  margin-left: 10px;
+  margin-right: 10px;
+  justify-content: flex-start;
+}
+
+.card-div-grey {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  margin-left: 10px;
+  margin-right: 10px;
+  justify-content: flex-start;
```

Comment:
shorthandd padding, 
shorthand margin
needs display flex for justify to work

Reply:
Thank you!
I removed the justify-content, it wasn't needed.